plugins/github-actions/src/components/useWorkflowRuns.ts

```diff
@@ -56,13 +57,51 @@ export function useWorkflowRuns({
   const [total, setTotal] = useState(0);
   const [page, setPage] = useState(0);
   const [pageSize, setPageSize] = useState(initialPageSize);
+  const [branches, setBranches] = useState<Branch[]>([]);
+  const [defaultBranch, setDefaultBranch] = useState<string>('');
 
   const {
     loading,
     value: runs,
     retry,
     error,
   } = useAsyncRetry<WorkflowRun[]>(async () => {
+    const fetchedDefaultBranch = await api.getDefaultBranch({
+      hostname,
+      owner,
+      repo,
+    });
+    setDefaultBranch(fetchedDefaultBranch);
+    if (branch === 'default') {
+      // eslint-disable-next-line no-param-reassign
+      branch = fetchedDefaultBranch;
```

Comment:
It's better to introduce a new variable then disabling the rule

Reply:
Fixed.